nautobot/core/tasks.py

```diff
@@ -1,15 +1,16 @@
-import logging
-
 from cacheops.simple import CacheMiss, cache
+from celery.utils.log import get_task_logger
+from celery.exceptions import TimeLimitExceeded
 from django.conf import settings
 from packaging import version
 import requests
 
 from nautobot.core import celery
 from nautobot.core.utils import config
 
+
 # Get an instance of a logger
-logger = logging.getLogger("nautobot.releases")
+logger = get_task_logger(__name__)
```

Comment:
What's the function of `get_task_logger` versus `logging.getLogger`?

Reply:
https://docs.celeryq.dev/en/stable/userguide/tasks.html#logging